index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Hola, ¿Como te llamas?');
+alert (`Hola ${nombreUsuario}!`);
```

Comment:
En este caso, me parece mejor que no uses interpolación de strings.
```suggestion
alert ('Hola ' + nombreUsuario + '!');
```